js/interface.js

```diff
@@ -575,18 +575,18 @@ function sortDataSources(key, order) {
     toBeOrderedDataSources = allDataSources;
   }
 
-  // Order by updatedAt
   var orderedDataSources = _.orderBy(toBeOrderedDataSources, function(ds) {
-    var date = new Date(ds[key]);
-    if (Object.prototype.toString.call(date) === "[object Date]" && !isNaN(date)) {
-      return new Date(ds[key]).getTime();
+    switch (key) {
+      case 'updatedAt':
+        return new Date(ds[key]).getTime();
+      case 'name':
+        var tempData = ds[key].toUpperCase();
```

Comment:
`temp` is not allowed as a word, please use a variable name which tells the reader what this is meant to do.

Reply:
@squallstar Fixed